### DISCUSSION TITLE
How to import naive-ui full-styles?

### DISCUSSION THREAD

**Discussion by QC2168** [tusen-ai/naive-ui#5403] [General]
I using unocss framework, May have caused style contamination, button style problems

**Comment by QC2168** [comment]
![image](https://github.com/tusen-ai/naive-ui/assets/82024018/ab9f90a0-a985-4398-9bfb-631689733fdf)

**Reply by Volankey** [reply]
See https://www.naiveui.com/en-US/os-theme/docs/style-conflict.

**Comment by 07akioni** [comment]
https://www.naiveui.com/en-US/os-theme/docs/style-conflict.
